apps/02-creacion-de-listas/src/create.erl

```diff
@@ -8,13 +8,30 @@
 %% @spec create(Value :: pos_integer()) -> [pos_integer()]
 %% @end
 %%--------------------------------------------------------------------
-create(_Value) ->
-    erlang:throw(not_implemented).
+create(N) when is_number(N)->
```

Comment:
`pos_integer()` is most restrictive than `is_number()`, you must change the guard, because the negative numbers is a valid value on `is_number()` but not in `pos_integer()`.

The same problem exists on `reverse_create()`